src/app/domain/sharing/models/SharingCozyApp.ts

```diff
@@ -0,0 +1,21 @@
+export interface SharingCozyApp {
+  _id: string
+  _type: string
+  accept_documents_from_flagship?: {
+    accepted_mime_types: string[]
+    max_number_of_files: number
+    max_size_per_file_in_MB: number
+    route_to_upload: string
+  }
+  accept_from_flagship?: boolean
+  attributes: {
+    accept_documents_from_flagship?: {
+      accepted_mime_types: string[]
+      max_number_of_files: number
+      max_size_per_file_in_MB: number
+      route_to_upload: string
+    }
+    accept_from_flagship?: boolean
+  }
+  slug: string
+}
```

Comment:
We should definitly stop adding types here but adding types to cozy-client. We have the type for the konnector https://github.com/cozy/cozy-client/blob/master/packages/cozy-client/types/types.d.ts#L47 we should add the one for the app. 

This current type miss a lot of stuff. You can find the all stuff here https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.apps.md and there https://docs.cozy.io/en/cozy-apps-registry/#properties-meaning-reference

By the way, you should update the documentation  in order to add the new stuff in there.